modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition_manager.py

```diff
@@ -42,6 +54,82 @@ class PandasOnRayDataframePartitionManager(GenericRayDataframePartitionManager):
     _execution_wrapper = RayWrapper
     materialize_futures = RayWrapper.materialize
 
+    if LazyExecution.get() in ("On", "Axis"):
```

Comment:
Whether to use this function or not is determined during the first import without the possibility of further replacement. As far as I remember, in all other places, functions are defined on each call.